modules/statics/src/coins.ts

```diff
@@ -16523,6 +16525,26 @@ export const coins = CoinMap.fromCoins([
     'xrpl.org',
     UnderlyingAsset['txrp:tst-rP9jPyP5kyvFRb6ZiRghAGw5u8SGAmU4bd']
   ),
+  suiToken(
+    'f26941b7-1110-4aa7-a2bc-29807297a51c',
+    'sui:deep',
+    'Deepbook',
+    6,
+    '0xdeeb7a4662eec9f2f3def03fb937a663dddaa2e215b8078a284d026b7946c270',
+    'deep',
```

Comment:
what does `module` mean?

Reply:
`0xdeeb7a4662eec9f2f3def03fb937a663dddaa2e215b8078a284d026b7946c270::deep::DEEP` is just a representation of `packageId::module::symbol`. The module & symbol name are defined in the source of the package. We need these since the rpc calls expect coin in `packageId::module::symbol` format & just having packageId wouldn't work